src/Icicle/Source/ToCore/ToCore.hs

```diff
@@ -80,8 +80,12 @@ convertQueryTop feats qt
                   Just t' -> return t'
         let inpTy'dated = T.PairT inpTy T.DateTimeT
 
-        (bs,ret) <- evalStateT (convertQuery $ query qt) (ConvertState inp inpTy'dated fs Map.empty)
-        let bs'   = strm inp C.Source <> bs
+        (now,bs,ret) <- evalStateT (do
```

Comment:
its ok @amosr  misery loves company and youtube vids are just so easy to trick people into watching....